testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/user/profile/CustomUserProfileTest.java

```diff
@@ -52,10 +54,12 @@ public void testCustomUserProfileProviderIsActive() {
     }
 
     private static void testCustomUserProfileProviderIsActive(KeycloakSession session) {
-        DeclarativeUserProfileProvider provider = getDynamicUserProfileProvider(session);
+        UserProfileProvider provider = getUserProfileProvider(session);
         assertEquals(CustomUserProfileProvider.class.getName(), provider.getClass().getName());
         assertTrue(provider instanceof  CustomUserProfileProvider);
-        assertEquals("custom-user-profile", provider.getComponentModel().getProviderId());
+        Optional<ComponentModel> component = getComponentModel(session);
```

Comment:
@vramik We need to make sure the configuration is set before checking the component. We can add this:

```
provider.setConfiguration(UPConfigUtils.readDefaultConfig());
```